packages/tsconfig.json

```diff
@@ -59,7 +59,6 @@
     {"path": "./react-serialize"},
     {"path": "./react-server"},
     {"path": "./react-server-webpack-plugin"},
-    {"path": "./react-shopify-app-route-propagator"},
```

Comment:
This was something we forgot to do when we deprecated `react-shopify-app-route-propagator`